airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/spec.py

```diff
@@ -105,17 +105,15 @@ class ConnectorConfig(BaseConfig):
     class Config:
```

Comment:
I assume this is generated, but my other comments to the spec.json apply here. Not sure which is generated from which